lib/web3/lp.js

```diff
@@ -0,0 +1,52 @@
+import { Web3Providers } from "./provider";
+import { BigNumber}  from "@ethersproject/bignumber";
+import { Contract } from "@ethersproject/contracts";
+import { initGeb } from "./geb";
+
+// Only Arbitrum for now because there aren't pools to query for other networks
+const geb = initGeb("ARBITRUM");
+
+const ERC20_ABI = [
+    'function balanceOf(address owner) view returns (uint256)',
+];
+
+const OD_ADDRESS = geb.tokenList['OD'].address
+const ODG_ADDRESS = geb.tokenList['ODG'].address
+
+const POOLS = {
+    od: [
+        { address: '0x824959a55907d5350e73e151Ff48DabC5A37a657', token: OD_ADDRESS }, // OD-ETH
+        { address: '0x0d867ef7cc5132881ff0216be9498a0993caaf68', token: OD_ADDRESS }  // OD-crvUSD
+    ],
+    odg: [
+        { address: '0xF935263c9950EB2881FF58Bd6a76c3D2564A78D5', token: ODG_ADDRESS }  // ODG-ETH
+    ],
+};
```

Comment:
Can you restructure this to avoid a situation where we pass the wrong token address for a pool address?

Reply:
Do you mean token symbol? OD_ADDRESS and ODG_ADDRESS are getting their address info from the geb object in the SDK (arbitrum network only), so I'm not sure in which case it would be wrong. 

If you meant passing in the wrong token symbol then if there is no pool for the specific token then this line will catch it:

```typescript
const pools = POOLS[tokenSymbol.toLowerCase()];

         if (!pools) {
             throw new Error(`No pools defined for token: ${tokenSymbol}`);
         }
```

Do you prefer if I hardcode the addresses of OD and ODG?

Reply:
@pi0neerpat Finished the changes where we do the lookup with the token address rather than the token symbol. I also added some more error handling for the geb initialization

Testing it works: 

<img width="707" alt="Screenshot 2024-05-22 at 3 34 13 PM" src="https://github.com/open-dollar/od-bot/assets/47253537/220eabde-b560-4ea4-8c2b-79700be889ac">

<img width="954" alt="Screenshot 2024-05-22 at 3 30 32 PM" src="https://github.com/open-dollar/od-bot/assets/47253537/74130bfa-c8e1-4eae-9977-92fd6711f54e">